virtManager/details/details.py

```diff
@@ -1935,7 +1935,8 @@ def config_vsock_apply(self, devobj):
                                           kwargs, self.vm, self.err,
                                           devobj=devobj)
 
-    def remove_devobj_internal(self, vm, err, devobj):
+    @staticmethod
+    def remove_devobj_internal(vm, err, devobj):
```

Comment:
Actually I think this function should be moved to vmmDeleteStorage class, then the callback doesn't need to be passed into the init function, and vmmDetails can call that function directly.

Put staticmethod functions at the top of the class definition, above the `__init__` function